site/docs/hosting/cyclic.md

```diff
@@ -0,0 +1,137 @@
+---
+prev: false
+next: false
+---
+
+# Hosting: Cyclic.sh
+
+This guide tells you about the ways you can host your grammY bots on [Cyclic.sh](https://cyclic.sh/), using Node.js.
+
+## Prerequisites
+
+To follow along, you need to have a [Github](https://github.com/) and [Cyclic.sh](https://cyclic.sh/) account.
+First, install some dependencies:
+
+```sh
+# Create a project directory.
+mkdir grammy-bot
+cd grammy-bot
+npm init --y
+
+# Install main dependencies.
+npm install grammy express
+
+# Install development dependencies.
+npm install -D typescript @types/express @types/node @types/node-fetch
```

Comment:
It should not be necessary to install types for node-fetch. Instead, enable skipLibCheck.